documentation/book/oauth/proc-oauth-server-config.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// assembly-oauth-strimzi-config.adoc
+
+[id='proc-oauth-server-config-{context}']
+= Deploying an {oauth} authorization server
+
+This procedure describes how to deploy an authorization server and configure it for integration with {ProductName}.
+
+Depending on the chosen xref:con-oauth-authentication-client-options-{context}[authentication flow], you can use the authorization server to configure authentication credentials for:
+
+* Configure brokers
+* Configure and authorize clients
+* Configure users
+* Obtain access and refresh tokens
+
+User credentials are managed by an authorization server using a _realm_, which describes security policies.
+The realm is the authentication provider for clients.
+
+To use {oauth} with {ProductName}, you need a deployment of an authorization server to be able to create and manage authentication realms.
+
+NOTE: If you already have an authorization server deployed, you can skip the deployment step and use your current deployment.
+
+.Prerequisites
+
+* {ProductName} and Kafka is running
+
+For the authorization server deployment:
+
+* Check the supported configuration for the authorization server
+* Installation requires a user with cluster-admin role, such as system:admin
+
+.Procedure
+
+. Deploy the authorization server to your cluster.
+. Access the CLI or admin console for the authorization server to create the {oauth} policies for {ProductName}.
++
+Consult the product documentation for the authorization server for information on creating the policies.
+
+. Create a _realm_.
++
+* Enable the realm
+* Set SSL as `external`
```

Comment:
I do not think this should be needed. This will be also Keycloak specific. I would remove this line.